CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
+if( NOT SKIP_GOOGLETEST_INSTALL )
+   set( ENABLE_GOOGLETEST_INSTALL ON )
```

Comment:
Okay; you were right about the option being set on the command line... while updating some other projects, I realized where the error is.  If you you do instead....
```
$ mkdir /tmp/cmake-option && mkdir /tmp/cmake-option/project && cd /tmp/cmake-option
$ echo 'set( FOO ON )\nadd_subdirectory(project)' > CMakeLists.txt
$ echo 'option(FOO "This is option FOO" OFF)' > project/CMakeLists.txt
$ cmake . 
$ grep FOO CMakeCache.txt
```
FOO will still be OFF.
So it doesn't work to do it that way for chained cmakelists.